vignettes/rromeo.Rmd

```diff
@@ -15,54 +15,166 @@ vignette: >
 ```{r setup, include = FALSE}
 library(vcr)
```

Comment:
Side note: the vcr dependency should probably have a minimal version number since it didn't work before the last update

Reply:
Fixed on master.